Cargo.toml

```diff
@@ -13,6 +13,7 @@ homepage = "https://github.com/cloud-hypervisor/cloud-hypervisor"
 lto = true
 
 [dependencies]
+api_client = { path = "api_client" }
```

Comment:
That name sounds a bit too generic. Don't you think something like `http_api_client` might be more appropriate?

Reply:
:thinking: Not conviced. I don't think we're ever going to have a non-HTTP API client...

Reply:
Fair enough, I guess we can still change that later if we need to.